src/obis-observer/commands/uplink/GetShortNamesResponse.ts

```diff
@@ -1,13 +1,12 @@
 import Command, {TCommandExampleList} from '../../Command.js';
-import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters, IObis} from '../../CommandBinaryBuffer.js';
+import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
 import {UPLINK} from '../../constants/directions.js';
 
 
 /**
- * IGetShortNameResponseParameters command parameters
+ * IGetShortNamesResponseParameters command parameters
```

Comment:
if `GetShortNamesResponse` exists - must be a downlink command `GetShortNames`

Reply:
fixed